dpu_utils/mlutils/vocabulary.py

```diff
@@ -86,7 +87,7 @@ def __str__(self):
     def get_all_names(self) -> FrozenSet[str]:
         return frozenset(self.token_to_id.keys())
 
-    def __batch_add_from_counter(self, token_counter: Counter, count_threshold: int, max_size:int) -> None:
+    def __batch_add_from_counter(self, token_counter: typing.Counter[str], count_threshold: int, max_size: int) -> None:

```

Comment:
This requires very recent versions of python (>= 3.6.1), as the older versions had no generic version of the Counter type.
Not sure this is an issue, but wanted to raise...

Reply:
I've made this explicit in the `setup.py`. I hope >=3.6.1 isn't an issue for anyone...